skyvern/forge/sdk/workflow/models/block.py

```diff
@@ -1141,13 +1142,18 @@ def is_safe_code(code: str) -> None:
         for node in ast.walk(tree):
             if hasattr(node, "attr") and str(node.attr).startswith("__"):
                 raise InsecureCodeDetected("Not allowed to access private methods or attributes")
-            if isinstance(node, ast.Import) or isinstance(node, ast.ImportFrom):
```

Comment:
this is definitely not production ready which is why we didn't remove the code.

We can have [ENABLE_CODE_BLOCK](https://github.com/Skyvern-AI/skyvern/blob/main/skyvern/config.py#L239) to guard the code block now. We can also make a config for supported builtin and libraries to import or import from.